src/traces/parcoords/lines.js

```diff
@@ -128,8 +126,8 @@ function makePoints(sampleCount, dims, color) {
         for(var k = 0; k < 64; k++) {
             points.push(
                 k < len ? dims[k].paddedUnitValues[j] :
-                    k === 64 - 1 ? adjustDepth(color[j]) :
-                        k >= 64 - 4 ? calcPickColor(j, 64 - 2 - k) : 0.5
+                    k === 63 ? adjustDepth(color[j]) :
+                        k > 59 ? calcPickColor(j, 62 - k) : 0.5
```

Comment:
Can we turn those `64`, `63`, `62` and `59` numbers into constants to make it easier to recognise? Thanks!

Reply:
This block is finally turned into https://github.com/plotly/plotly.js/blob/75f692dda8b6905d6399cb05fc09ff9ab63fff56/src/traces/parcoords/lines.js#L119-L125
where the first 60 elements are position data and last four are dedicated to color data.

Reply:
Nicely done!